code/modules/projectiles/magazines/lever_action.dm

```diff
@@ -74,6 +74,28 @@ Similar to shotguns.dm but not exactly.
 	max_rounds = 9
 	chamber_closed = 0
 
+// This is the internal magazine of the spotter XM rifle
+/obj/item/ammo_magazine/internal/lever_action/xm88/spotter
+	name = "XM88 spotter rifle internal magazine"
+	desc = "If you see this something went wrong and the world is ending!!"
+	icon = 'icons/obj/items/weapons/guns/ammo_by_faction/uscm.dmi'
+	icon_state = "458-box"
+	item_state = "458-box"
+	default_ammo = /datum/ammo/bullet/lever_action/tracker
+	caliber = ".458 tracker"
+	gun_type = /obj/item/weapon/gun/lever_action/xm88/spotter
+	max_rounds = 1
+
+/obj/item/ammo_magazine/internal/lever_action/xm88/spotter/Initialize(mapload, spawn_empty)
+	. = ..()
+	src.color = "#FFCC00" // Yes recoloring things is the new coder sprites what about it
+
+// This si the box that comes in the spotter kit
+/obj/item/ammo_magazine/internal/lever_action/xm88/spotter/box
```

Comment:
Don't use internal magazines for this, these are meant for internal firearms usage (holding rounds as a revolver barrel or shotgun tube, etc) and not as usable items

Reply:
Funny enough the reason that I put that in there is intentional if I remove the internal it causes the box to spawn fine but the rounds inside the box will be the SOCOM rounds not the tracer rounds.

Maybe some kind of bug with how the parents over-write ammo box contents but the way I solved this was to change the parent before we get to lever_action and "internal" works for my needs.

![dreamseeker_7hcTAkPHf7](https://github.com/cmss13-devs/cmss13/assets/24533979/acefd32f-f20f-46cd-8da8-bc3b9379c384)

Reply:
See `/obj/item/ammo_magazine/lever_action/tracker`. If you inherit `/obj/item/ammo_magazine/lever_action/xm88` then of course its going to contain `/datum/ammo/bullet/lever_action/xm88` unless you alter `default_ammo`